src/twisted/topfiles/8781.bugfix

```diff
@@ -0,0 +1 @@
+Client and server TLS connections made via the client TLS endpoint and the server SSL endpoint, as well as any other code that uses twisted.internet.ssl.CertificateOptions, no longer accept 3DES-based cipher suites, to defend against SWEET32.
```

Comment:
:+1:

On Wed, Aug 24, 2016 at 10:35 AM, Cory Benfield <notifications@github.com>
wrote:

> In src/twisted/topfiles/8781.bugfix
> <https://github.com/twisted/twisted/pull/522#discussion_r76068266>:
>
> > @@ -0,0 +1 @@
> > +Client and server TLS connections made via the client TLS endpoint and the server SSL endpoint, as well as any other code that uses twisted.internet.ssl.CertificateOptions, no longer accept 3DES-based cipher suites, to defend against SWEET32.
>
> Done.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/twisted/twisted/pull/522/files/dc9f245661f6e43c42c4db2773af0677e8392ea6#r76068266>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAADBNdDhpbh2737DZ8x-RNuDPa3vq-Fks5qjFaagaJpZM4JsCSY>
> .
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: D1B3 ADC0 E023 8CA6